apps/api/src/app/models/permissions/permissions.schema.ts

```diff
@@ -0,0 +1,22 @@
+import { Schema } from 'mongoose';
+import { ViewModels } from '../../helpers/constants';
+
+export const PermissionsSchema = new Schema(
+  {
+    user: {
+      _id: { type: String, ref: ViewModels.USERS_VIEW },
+      name: String,
+    },
+    role: {
+      type: String,
+      max: 100,
+      required: true,
+    },
+    categories: [],
```

Comment:
inside array should add type, bellow same

Reply:
Categories and Lessons features don't exist.